model/task/task_test.go

```diff
@@ -3509,24 +3509,29 @@ func TestArchiveMany(t *testing.T) {
 	tasks := []Task{t1, t2, dt}
 	err := ArchiveMany(ctx, tasks)
 	assert.NoError(t, err)
-	currentTasks, err := FindAll(db.Query(ByVersion("v")))
-	assert.NoError(t, err)
-	assert.Len(t, currentTasks, 4)
-	for _, task := range currentTasks {
-		assert.False(t, task.Aborted)
-		assert.Equal(t, 1, task.Execution)
-	}
-	oldTasks, err := FindAllOld(db.Query(ByVersion("v")))
-	assert.NoError(t, err)
-	assert.Len(t, oldTasks, 4)
-	for _, task := range oldTasks {
-		assert.True(t, task.Archived)
-		assert.Equal(t, 0, task.Execution)
+	verifyTasksFunc := func() {
```

Comment:
nit: it's usually more idiomatic to call this `verifyTasks` (or maybe something descriptive like `verifyTasksRestarted`) since it's already pretty clear that it's a function.

Reply:
I'll go with that, thanks